### ISSUE TITLE
[Quest] Captain Tyralius

### ISSUE THREAD

**Issue by Cone182** [dalaranwow/dalaran-wow#2074]
link: http://www.wowhead.com/quest=10422/captain-tyralius

Issue:
- Missing cklickable/useable object.

How it works:
- The key can be optained from the warden, but It cannot be used on Captain Tyralius´s prison.

How it should work:
- You should be able to use the key you obtained from the warden on the proson of Captain tyralius.

**Comment by Rushor** [comment]
```sql
UPDATE `gameobject_template` SET `data2`=180000 WHERE `entry`=184588;
```

**Comment by MrAndersonX** [comment]
Confirmed - still bugged.

**Comment by jonanone** [comment]
The prison now closes automatically on a given time so it can be opened once again. The fix will be applied to the live realm soon.

**Comment by Golganeth** [comment]
The prison could use a shorter respawn time (7-8 minutes roughly right now on PTR).

I completed the quest just fine once. But when the Prison+Tyralius respawned, the nearby NPCs started attacking him (Ethereum Nexus-Stalker). Once the captain died I killed the stalkers and opened the prison, no credit was given.

Tyralius shouldn't be attackable by nearby NPCs nor players. I'm assuming something goes wrong with Tyralius' flag once it respawns, because it didn't happen the first time around.
